app/posts/manage-breast-screening/2025/05/2025-05-06-image-reading-first-prototype.md

```diff
@@ -0,0 +1,112 @@
+
+Once prepared, staff place clinic folders on shelves in date order, separated by first and second read status. Radiologists typically select the oldest clinic, though they might prioritise second reads if an arbitration meeting is approaching.
+
+### Why read in a batch?
+
+Processes have evolved over time to be highly reliant on reading in batches. Breast Screening Units (BSUs) track their participants via their screening forms and these are grouped into batches by clinic.
+
+There are processes and safeguards in place to track these clinics and make sure they move through screening in a timely manner and that no individuals get lost. It’s naturally easier to track a clinic as a whole than every participant individually. If you’re tracking work via a clinic, it makes sense to do reading by clinic.
+
+Reading in clinic batches has become a useful unit of work. A clinic will typically have 30 to 60 cases in it, and a radiologist might expect to complete this in an hour. Prior studies around reading order and reader fatigue have found evidence around suggested reading session durations. Research has also identified benefits of there being a defined unit of work someone can work through to ‘finish’.
+
+Reading in batches ultimately meets two core user needs. Image readers firstly need to read the right cases in the right order so that nothing gets missed, and secondly they need to know how many cases to read or how long to read for in one go, so that they can break work down into manageable chunks and have an end point to aim for.
+
+## Image reading dashboard
+
+Our first prototype largely recreates the current workflow of reading by batch. We list clinics in reverse-date order and show their reading status. This will need to be more sophisticated in the future when we address first vs. second reads and situations where a clinic may be partially first read and partially second read.
+
+We've included a 'Start reading' link at the top of the page – this links directly to the oldest clinic available for reading. We're interested to see whether radiologists tend to start reading immediately or prefer reviewing the clinics first to select one to read.
+
+![A screenshot of our image reading dashboard](dashboard.png)
+
+When you select a clinic or ‘start reading’ it takes you through to a page showing a list of cases in the clinic and their read status. Again, users have a choice between ‘start reading’ or to select a case.
+
+![A list of cases in a clinic to be read](clinic-list.png)
+
+## Giving an opinion
+
+When a radiologist opens a case, we show high-level information about the participant with buttons to give an opinion. This works similarly to the National Breast Screening System (NBSS) – when opening a case, the relevant mammograms open in the radiologist's Picture Archiving and Communication System (PACS) image viewer. Our job is to let them record their opinion and review any relevant medical information.
```

Comment:
Is this 2 sentences? Or is the second part explaining how it works in NBSS, which case a colon might be clearer?

```suggestion
When a radiologist opens a case, we show high-level information about the participant with buttons to give an opinion. This works similarly to the National Breast Screening System (NBSS). When opening a case, the relevant mammograms open in the radiologist's Picture Archiving and Communication System (PACS) image viewer. Our job is to let them record their opinion and review any relevant medical information.
```